ptbcontrib/username_to_chat_api/requirements.txt

```diff
@@ -1 +1,2 @@
-python-telegram-bot>=13.0,<20
\ No newline at end of file
+python-telegram-bot>=20
```

Comment:
`pip install ptb>=20 --pre` doesn't match 20.0a0. for the others, I just explicitly specified 20.0a0. During the pre-release phase, that might be a bit safer …

Reply:
right.
```suggestion
python-telegram-bot==20.0a0
```